src/couch_replicator/src/couch_replicator_api_wrap.erl

```diff
@@ -284,7 +284,7 @@ open_doc_revs(#httpdb{retries = 0} = HttpDb, Id, Revs, Options, _Fun, _Acc) ->
         couch_replicator_httpc:full_url(HttpDb, [{path, Path}, {qs, QS}])
     ),
     couch_log:error("Replication crashing because GET ~s failed", [Url]),
-    exit(kaboom);
+    exit(open_doc_revs_failed);
```

Comment:
it would be neat if we could report at least the most recent _reason_ for the failure (of the last retry) either to the log or in the response body, or both!

Reply:
Good idea. I'll try updating to crash before recursing the last time. Once we recurse with retries=0 we already lost the error reason.

Reply:
Updated to log the error with couch_log:error/2. Was a bit hesitant to use that in the exit atom as we pattern match on error results in a few cases and we also end up with a large stack trace in there as well.